File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/SKPoint.java
package br.com.insanitech.spritekit;

import br.com.insanitech.spritekit.opengl.model.GLPoint;

/**
 * Created by anderson on 7/4/15.
 */
public class SKPoint extends GLPoint {
    public SKPoint() {
        super();
    }

    public SKPoint(float x, float y) {
        super(x, y);
    }

    @Override
    public String toString() {
        return "{ x: " + getX() + ", y: " + getY() + " }";
    }
}

File: SpriteKitLib/src/main/java/br/com/insanitech/spritekit/SKPoint.kt
package br.com.insanitech.spritekit

import br.com.insanitech.spritekit.opengl.model.GLPoint

/**
 * Created by anderson on 7/4/15.
 */
class SKPoint : GLPoint {
    constructor() : super() {}

    constructor(x: Float, y: Float) : super(x, y) {}

    constructor(other: SKPoint) {
        x = other.x
        y = other.y
    }

    override fun toString(): String = "{ x: $x, y: $y }"
}
